pkg/commands/drift/command.go

```diff
@@ -85,6 +89,48 @@ func (c *DetectIamDriftCommand) Flags() *cli.FlagSet {
 		Usage:   `The maximum number of concurrent requests allowed at any time to GCP.`,
 		Default: 10,
 	})
+	f.BoolVar(&cli.BoolVar{
+		Name:    "skip-github-issue",
+		Target:  &c.flagSkipGitHubIssue,
+		Example: "true",
+		Usage:   `Whether or not to create a GitHub Issue when a drift is detected.`,
+		Default: false,
+	})
+	f.StringVar(&cli.StringVar{
+		Name:    "gh-token",
+		Target:  &c.flagGitHubToken,
+		Example: "...",
```

Comment:
Remove this instead of ambiguous ... throughout.

Reply:
Done